templates/create-network.html

```diff
@@ -19,6 +19,12 @@ <h4 class="modal-title">Create Network</h4>
     <tr>
       <td>Genes</td>
       <td><input type='text' id='geneRegex' class="form-control"></td>
+      <td>
+        <div id='gene-input-type' class="input-checkbox">
```

Comment:
Please use the name and order of radioboxes "text/regex" to match other parts of the system.